api/src/main/java/io/opencensus/metrics/export/TimeSeries.java

```diff
@@ -63,6 +63,28 @@ public static TimeSeries create(
         startTimestamp);
   }
 
+  /**
+   * Creates a {@link TimeSeries}.
+   *
+   * @param labelValues the {@code LabelValue}s that uniquely identify this {@code TimeSeries}.
+   * @param point the single data {@code Point} of this {@code TimeSeries}.
+   * @param startTimestamp the start {@code Timestamp} of this {@code TimeSeries}. Must be non-null
+   *     for cumulative {@code Point}s.
+   * @return a {@code TimeSeries}.
+   * @since 0.17
+   */
+  public static TimeSeries createWithOnePoint(
+      List<LabelValue> labelValues, Point point, @Nullable Timestamp startTimestamp) {
+    // Fail fast on null lists to prevent NullPointerException when copying the lists.
```

Comment:
Consider using `create(labelValues, Collections.singletonList(point), timestamp);` instead of copying the whole code block.

Reply:
The whole idea is to avoid:
Collections.unmodifiableList(new ArrayList<Point>(Collections.singletonList(point)));

I think we should have a createInternal version probably to avoid duplicate.

Reply:
SGTM